dbt_sugar/core/task/audit.py

```diff
@@ -33,15 +33,17 @@ def __init__(self, flags: FlagParser, dbt_path: Path, sugar_config: DbtSugarConf
     def run(self) -> int:
         """Main script to run the command doc"""
         if self.model_name:
+            _ = self.is_exluded_model(self.model_name)
             logger.info(f"Running audit of model [bold magenta]{self.model_name}.[/bold magenta]\n")
-            path_file, schema_exists = self.find_model_in_dbt(self.model_name)
+            path_file, schema_exists = self.find_model_schema_file(self.model_name)
             if not path_file:
                 logger.info(f"Could not find {self.model_name} in the project at {self.dbt_path}")
                 return 1
             if not schema_exists:
                 logger.info("The model is not documented.")
                 return 1
             self.model_content = open_yaml(path_file)
+            logger.debug(f"Content for '{self.model_name}' - audit_task: {self.model_content} ")
```

Comment:
This self.model_content is going to be all the schema.YAML file content, imagine a super big model the result will be super big print it can even crash the console.

Reply:
good catch going to change that